content/blog/java-1-0/index.md

```diff
@@ -0,0 +1,91 @@
+---
+title: "Pulumi's Java SDK is Now Generally Available"
```

Comment:
Can we have a little more fun w. the title?

"Our Java Tastes Better Than Ever"

or anything that looks like we have some personality?  :)